tensorflow/core/kernels/mkl/mkl_conv_ops.cc

```diff
@@ -1556,77 +1584,233 @@ class MklQuantizedConv2DOp
     }
   }
 
-  explicit MklQuantizedConv2DOp(OpKernelConstruction* context)
-      : MklConvOp<Device, Tinput, qint8, Tbias, Toutput, Ttemp_output, int32,
-                  bias_enabled, false, is_depthwise, native_format>(context) {
-    bool is_filter_const;
-    OP_REQUIRES_OK(context,
-                   context->GetAttr("is_filter_const", &is_filter_const));
+  explicit MklQuantizedConvOp(OpKernelConstruction* context)
+      : MklConvOp<Device, Tinput, /*Tfilter*/ qint8, Tbias, Toutput,
+                  Ttemp_output, /*Tpadding*/ int32,
+                  /*bias_enabled*/ false, /*pad_enabled*/ false, is_depthwise,
+                  /*native_format*/ true>(context) {
+    std::vector<std::vector<string>> supported_fusions = {
+        {"BiasAdd"},
+        {"Relu"},
+        {"Requantize"},
+        {"BiasAdd", "Relu"},
+        {"BiasAdd", "Requantize"},
+        {"Relu", "Requantize"},
+        {"BiasAdd", "Relu", "Requantize"},
+        {"BiasAdd", "Sum", "Relu"},
+        {"BiasAdd", "Sum", "Relu", "Requantize"}};
```

Comment:
I fully agree that storing the list of supported fusions as a `vector<int64_t>` would provide faster lookup compared to `vector<vector<string>>`.

Since we plan to phase out the old API, would it be okay to implement this change when the old API is fully removed? The reason is because this would require changes to the op definition and all the unit tests for both the new and old APIs. I think it would be much easier to make this change for just the new API. Also, with the TF 2.10 cutoff date being so close, I am not sure if we will have sufficient time to test this change. 

Please let me know what you think.

Reply:
I agree there isn't enough time to make the change and fully test it before TF 2.10 branch cut. Let's do it after the branch cut. Sorry again for my delay!